libs/sdk-ui-kit/styles/scss/button.scss

```diff
@@ -595,14 +600,14 @@
 
     > .gd-button:first-child,
     > *:not(.gd-button):first-child .gd-button {
-        @include border-radius(3px 0 0 3px);
+        border-radius: $button-group-childs-border-radius;
     }
 
     > .gd-button:last-child,
     > *:not(.gd-button):last-child .gd-button {
         margin-right: 0;
 
-        @include border-radius(0 3px 3px 0);
+        border-radius: $button-group-childs-border-radius;
```

Comment:
this should not be single variable. It should be here like :
 0 $button-border-radius $button-border-radius 0